Mission.md

```diff
@@ -0,0 +1,5 @@
+# Mission Statement
+
+The Planetary Software Organization promotes open source software in the planetary sciences by helping software creators and maintainers foster an open source community.
+
+The Planetary Software Technical Steering Committee (TSC) helps software creators grow their community by providing community guidelines and standards. The TSC also helps software maintainers manage the growth and evolution of open source software by providing an open source governance model.
```

Comment:
"Planetary Software Organization Techinical..." ?

Reply:
Just the naming standard as per the README in our repo. I think consistency would be key here.

Reply:
Fair point.